tests/integration/pilot/common/routing.go

```diff
@@ -120,6 +120,8 @@ func virtualServiceCases(skipVM bool) []TrafficTestCase {
 	})
 	var cases []TrafficTestCase
 	cases = append(cases,
+		// Retry conditions have been added to just validate that config is correct.
```

Comment:
should we just add another case? This makes the "Added header" one more complicate

Reply:
Yes. Thanks.  as usual your idea makes lot of sense :-). Moved.